assembl/graphql/schema.py

```diff
@@ -186,17 +333,60 @@ class Meta:
         only_fields = ('id', )
 
     title = graphene.String(lang=graphene.String())
-    posts = graphene.List(PropositionPost, random=graphene.Boolean())  # TODO should be a real ConnectionField
+    title_entries = graphene.List(LangStringEntry)
```

Comment:
Good question, I didn't thought of that. I wrote the title stuff and then the title_entiries with LangStringEntry. I think it's simpler on the frontend side to just have title which is just a string and not an object litteral with value and localeCode.